users/models.py

```diff
@@ -0,0 +1,74 @@
+""" This module contains the models for the users app.
+"""
+
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+# class Address(models.Model):
+#     """a model for the address of the user."""
+
+#     Country = models.CharField(max_length=50)
+#     city = models.CharField(max_length=50)
+#     address = models.CharField(max_length=50)
+#     postal_code = models.CharField(max_length=10)
+
+#     class Meta:
+#         db_table = "addresses"
+#         verbose_name = "Address"
+#         verbose_name_plural = "Addresses"
+
+#     def __str__(self):
+#         return self.address + ", " + self.city + ", " + self.Country
+
+
+class AdminsManager(models.Manager):
```

Comment:
I prefer it should be singular 
```suggestion
class AdminManager(models.Manager):
```